### ISSUE TITLE
Task "Update root password for all root accounts" fails when hostname is "Default"

### ISSUE THREAD

**Issue by janimattiellonen** [phansible/phansible#165]
```
failed: [192.168.111.252] => (item=Default) => {"failed": true, "item": "Default", "parsed": false}
SUDO-SUCCESS-dqzkppcwgiozzjgcdwzzkixcxyrdvcbk
Traceback (most recent call last):
  File "/home/vagrant/.ansible/tmp/ansible-tmp-1420308966.54-54310281768850/mysql_user", line 2204, in <module>
    main()
  File "/home/vagrant/.ansible/tmp/ansible-tmp-1420308966.54-54310281768850/mysql_user", line 498, in main
    changed = user_add(cursor, user, host, password, priv)
  File "/home/vagrant/.ansible/tmp/ansible-tmp-1420308966.54-54310281768850/mysql_user", line 177, in user_add
    cursor.execute("CREATE USER %s@%s IDENTIFIED BY %s", (user,host,password))
  File "/usr/lib/python2.7/dist-packages/MySQLdb/cursors.py", line 174, in execute
    self.errorhandler(self, exc, value)
  File "/usr/lib/python2.7/dist-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
_mysql_exceptions.OperationalError: (1396, "Operation CREATE USER failed for 'root'@'default'")
```

It seems to work with "default". I inspected the mysql.user table inside the vagrant box and there the hostname is "default", not "Default". Maybe this is some kind of case sensitivity issue.

**Comment by naxhh** [comment]
This was fixed in master.
https://github.com/phansible/phansible/pull/130

I'll do the changes for 1.0 and I'll ask Erika to deploy this.

Thanks for reporting!

**Comment by naxhh** [comment]
Hmm I was wrong... this was already fixed in #130 for 1.0

But seems is not deployed in Phansible website. @erikaheidi can you check this?
